src/fontra/client/core/utils.js

```diff
@@ -221,3 +221,23 @@ export function makeUPlusStringFromCodePoint(codePoint) {
     ? "U+" + codePoint.toString(16).toUpperCase().padStart(4, "0")
     : "";
 }
+
+export function buildShortcut(shorCutRegister) {
+  let shorcutCommand = "";
+
+  if (shorCutRegister) {
+    const isMac = navigator.platform.toLowerCase().indexOf("mac") >= 0;
+
+    if (shorCutRegister.shiftKey) {
+      shorcutCommand += isMac ? "&#8679;" : "Shift+"; // ⇧ or Shift
+    }
+    if (shorCutRegister.metaKey) {
+      shorcutCommand += isMac ? "&#8984;" : "Ctrl+"; // ⌘ or Ctrl
+    }
+    if (shorCutRegister.keysOrCodes) {
+      shorcutCommand += shorCutRegister.keysOrCodes.toUpperCase();
```

Comment:
While this works for the current cases, `keysOrCodes` is meant to accept either a string of multiple single chars (in that case, take the first), or a list of key codes. In the latter case, take the first item of the list, but this will be a string like "ArrowUp", and we need to see how to represent the key code.

The "Delete" use case will be a nice test. I suggest to leave this as it is for now, but count on having to change it for the "Delete" short cut.

Reply:
Mhh, maybe we can create some dictionary with definition and representation on different platforms and just look for them when we need it. E.g
```
const shortcutDefinitions = [
  {
    key: "ArrowUp",
    specialChar: "↑", // Used on a Mac or Windows but when it's available
    fallbackChar: "&#8593;", // Used as a fallback
  },
  {
    key: "Delete",
    specialChar: "⌫",
    fallbackChar: "&#9003;",
  },
  {
    key: "Home",
    specialChar: "", // Empty slot when it's not available
    fallbackChar: "Home", // Used as a fallback
  },
];

// To access a key we can browse it like this:
shortcutDefinitions.find(shortcut => shortcut.key === shorCutRegister.keysOrCodes);
```

Reply:
Something like that sounds like a good idea.

I think `"↑"` is 100% equivalent to `"&#8593;"`, it's just a different way of spelling it: unicode vs html entity. Modern browsers should all be able to display such chars, and it may be more a matter of following the platform conventions of how short cuts are displayed.